scaletest/workspacetraffic/run.go

```diff
@@ -160,16 +168,17 @@ func (r *Runner) Run(ctx context.Context, _ string, logs io.Writer) (err error)
 
 	var waitCloseTimeoutCh <-chan struct{}
 	deadlineCtxCh := deadlineCtx.Done()
+	wchRef, rchRef := wch, rch
```

Comment:
Hm... I think I got lost with `rch` and `rchRef` juggling. Was there any bug that you had to fix it?

Reply:
I refactored it to be safer because we were shadowing inside a goroutine, and reassigning on the "main" routine, meaning if the goroutine didn't start in time it could be racy. Mostly only relevant if something went wrong at startup, but still.

Reply:
I see, thanks!